### PR TITLE
Update docker-compose files

### PR DESCRIPTION
**Pull request by ZhmakAS**
Actualizing docker-compose files with new versions of binaries stored on Github registry. Updated docker/acra-build.dockerfile with ability to compile binary files with native Golang `netgo` DNS resolver.

Corresponding changes displayed in CHANGELOG_DEV.md.

- [+] Change is covered by automated tests
- [+] The [coding guidelines] are followed
- [+] Public API has proper documentation in the [Acra documentation] site or has PR on [documentation repository] 
  with new changes
- [+] CHANGELOG.md is updated (in case of notable or breaking changes)
- [+] CHANGELOG_DEV.md is updated
- [-] Benchmark results are attached (if applicable)
- [+] [Example projects and code samples] are up-to-date (in case of API changes) 

[coding guidelines]: https://golang.org/doc/effective_go
[Example projects and code samples]: https://github.com/cossacklabs/acra-engineering-demo
[Acra documentation]: https://docs.cossacklabs.com/
[documentation repository]: https://github.com/cossacklabs/product-docs

### CODE REVIEW

**Review by vixentael — APPROVED** [review]

File: docker/acra-build.dockerfile [hunk 1]
```diff
@@ -73,7 +73,7 @@ ENV PATH="$PATH:/usr/local/lib/go/latest/bin"
 ENV PATH_ACRA="/acra"
 COPY ./ "${PATH_ACRA}/"
 # Fetch all dependencies and build all binaries in acra
-RUN cd "${PATH_ACRA}" && go install ./cmd/...
+RUN cd "${PATH_ACRA}" && go install -tags netgo ./cmd/...
```
Comment (vixentael) [inline docker/acra-build.dockerfile:76]: @shadinua ^

Comment (ZhmakAS) [reply docker/acra-build.dockerfile:76]: Need additional testing from @shadinua

**Comment by Lagovas** [comment]
I propose to leave this PR open until @shadinua confirms that all is okay with other cases of resolving and after that merge. To not merge not tested configurations. To not revert changes after that with new one PR)

**Review by Lagovas — APPROVED** [review]

**Review by shadinua — APPROVED** [review]

File: docker/acra-build.dockerfile (hunk 1 above)
Comment (shadinua) [reply docker/acra-build.dockerfile:76]: Tested with scratch and Debian-based images. Thanks!

**Pull request merged**